code/modules/clothing/clothing.dm

```diff
@@ -16,6 +16,12 @@
 	//Used for hardsuits. If false, this piece cannot be retracted while the core module is engaged
 	var/retract_while_active = TRUE
 
+/obj/item/clothing/Initialize(mapload, ...)
+	. = ..()
+	if(!matter)
+		matter = list()
+	matter.Add(list(MATERIAL_BIOMATTER = 5 * w_class))	// based of item size
```

Comment:
But why?

Reply:
i was asked to do it, need more stuff with biomatter